src/navigation/rootNavigation.js

```diff
@@ -0,0 +1,46 @@
+// @flow
+
+import * as React from "react";
+import { NavigationContainer } from "@react-navigation/native";
+import { createDrawerNavigator } from "@react-navigation/drawer";
+
+import PlaceholderComponent from "../components/PlaceholderComponent";
+import ObservationProvider from "../providers/ObservationProvider";
+import StackNavigator from "./stackNavigation";
+
+// this removes the default hamburger menu from header
+const screenOptions = { headerLeft: ( ) => <></> };
+const hideHeader = {
+  headerShown: false,
+  label: "my observations"
+};
+
+const Drawer = createDrawerNavigator( );
+
+const App = ( ): React.Node => (
+  <NavigationContainer>
+    <ObservationProvider>
+      <Drawer.Navigator screenOptions={screenOptions} name="Drawer">
+        <Drawer.Screen
+          name="my observations"
+          component={StackNavigator}
```

Comment:
If this is a StackNavigator that *only* manages the stack within My Observations, shouldn't we give it a unique name, like `MyObservationsStackNavigator`? Or is there a way we could reuse it for any drawer?